executor/executor.go

```diff
@@ -257,17 +257,26 @@ func (e *ShowDDLExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 	}
 
 	ddlJobs := ""
+	query := ""
 	l := len(e.ddlInfo.Jobs)
 	for i, job := range e.ddlInfo.Jobs {
 		ddlJobs += job.String()
+		query += job.Query
 		if i != l-1 {
 			ddlJobs += "\n"
+			query += "\n"
 		}
 	}
+
+	cfg := config.GetGlobalConfig()
+	serverIP := cfg.AdvertiseAddress
+
 	chk.AppendInt64(0, e.ddlInfo.SchemaVer)
 	chk.AppendString(1, e.ddlOwnerID)
-	chk.AppendString(2, ddlJobs)
-	chk.AppendString(3, e.selfID)
+	chk.AppendString(2, serverIP)
```

Comment:
Should we show `serverIP:port` here in case of multiple tidb servers run on the same machine?

Reply:
That's ok.